ops/services/app_functions/report_stream_batched_publisher/functions/QueueBatchedReportStreamUploader/index.test.ts

```diff
@@ -21,6 +24,7 @@ jest.mock("../config", () => ({
     AZ_STORAGE_ACCOUNT_NAME: "hola",
     AZ_STORAGE_ACCOUNT_KEY: "bonjour",
     TEST_EVENT_QUEUE_NAME: "ciao",
+    FILE_ERROR_QUEUE_NAME: "cheese",
```

Comment:
lol I like how all the other mocks are hello in different languages and then.. 🧀